test/unit/visitor/after_cvode_to_cnexp.cpp

```diff
@@ -0,0 +1,68 @@
+/*************************************************************************
+ * Copyright (C) 2018-2020 Blue Brain Project
+ *
+ * This file is part of NMODL distributed under the terms of the GNU
+ * Lesser General Public License. See top-level LICENSE file for details.
+ *************************************************************************/
+
+#include <catch/catch.hpp>
+
+#include "ast/program.hpp"
+#include "parser/nmodl_driver.hpp"
+#include "test/unit/utils/test_utils.hpp"
+#include "visitors/after_cvode_to_cnexp_visitor.hpp"
+#include "visitors/checkparent_visitor.hpp"
+#include "visitors/nmodl_visitor.hpp"
+#include "visitors/symtab_visitor.hpp"
+
+
+using namespace nmodl;
+using namespace visitor;
+using namespace test;
+using namespace test_utils;
+
+using nmodl::parser::NmodlDriver;
+
+
+//=============================================================================
+// AfterCVodeToCnexp visitor tests
+//=============================================================================
+
+std::string run_after_cvode_to_cnexp_visitor(const std::string& text) {
+    NmodlDriver driver;
+    const auto& ast = driver.parse_string(text);
+
+    SymtabVisitor().visit_program(*ast);
+    AfterCVodeToCnexpVisitor().visit_program(*ast);
+    std::stringstream stream;
+    NmodlPrintVisitor(stream).visit_program(*ast);
```

Comment:
just a nitpick : I think we have `to_nmodl(ast)` the tdoes the same?

Reply:
You're right. `to_nmodl()` just calls underneath the `NmodlPrintVisitor()`. I can use `to_nmodl()` here to make it more readable maybe